awx_collection/plugins/modules/export.py

```diff
@@ -137,6 +137,20 @@
 def main():
     argument_spec = dict(
         all=dict(type='bool', default=False),
+        organizations=dict(type='list', elements='str'),
```

Comment:
This is confusing, how did this work before?

Reply:
I believe ansible will just inject data from the yaml file, regardless of what is in the argument_spec. But explicitly defining argument_spec allows for better data validation at runtime, so it should be set. That is my understanding anyways

Reply:
`ansible-test` should have been validating this for years, so I'm not sure why it's only failing now

Reply:
@gundalow it has been but I think as of late, ansible has allowed a larger arg_spec to be defined so we might have been ignoring the error (not uncommon).